lib/routes/account.js

```diff
@@ -491,6 +491,11 @@ module.exports = (log, db, mailer, Password, config, customs, checkPassword, pus
                   // deleted the original email they signed up with
                   return db.emailRecord(email)
                     .then((emailRecord) => {
+                      if (originalLoginEmail === undefined) {
+                        // User deleted original email and attempted to login with it,
+                        throw error.cannotLoginWithSecondaryEmail()
```

Comment:
Well spotted, there wasn't a test for this. Now that I think about it, this should also be a new error code as well.